modules/mobile/app/services/mobile/v2/appointments/proxy.rb

```diff
@@ -24,6 +24,11 @@ def get_appointments(start_date:, end_date:, include_pending:, pagination_params
           filterer = PresentationFilter.new(include_pending:)
           appointments = appointments.keep_if { |appt| filterer.user_facing?(appt) }
 
+          appointments = appointments.each do |appointment|
```

Comment:
Logic itself looks good but I think this can be moved into the `vaos_v2_to_v0_appointment_adapter.parse` method below since it's already looping through the appts there. looks something like: 
``` 
def build_appointment_model
          adapted_appointment = {
            id: appointment[:id],
            appointment_type:,
            cancel_id:,
            comment:,
            facility_id:,
            sta6aid: facility_id,
            healthcare_provider: appointment[:healthcare_provider],
            healthcare_service:,
            location:,
            minutes_duration: minutes_duration(appointment[:minutes_duration]),
            phone_only: appointment[:kind] == PHONE_KIND,
            start_date_local:,
            start_date_utc:,
            status:,
            status_detail: cancellation_reason(appointment[:cancelation_reason]),
            time_zone: timezone,
            vetext_id:,
            reason:,
            is_covid_vaccine: appointment[:service_type] == COVID_SERVICE,
            is_pending: requested_periods.present?,
            proposed_times:,
            type_of_care: type_of_care(appointment[:service_type]),
            patient_phone_number:,
            patient_email:,
            best_time_to_call: appointment[:preferred_times_for_phone_call],
            friendly_location_name:,
            service_category_name: appointment.dig(:service_category, 0, :text),
            appointment_ien: extract_station_and_ien(appointment)

          }

          StatsD.increment('mobile.appointments.type', tags: ["type:#{appointment_type}"])

          Mobile::V0::Appointment.new(adapted_appointment)
        end
        # rubocop:enable Metrics/MethodLength

        private

        def extract_station_and_ien(appointment)
          return nil if appointment[:identifier].nil?

          regex = %r{VistADefinedTerms/409_(84|85)}
          identifier = appointment[:identifier].find { |id| id[:system]&.match? regex }

          return if identifier.nil?

          identifier[:value]&.split(':', 2)&.second
        end```